build-scripts/gen_audio_resampler_filter.c

```diff
@@ -41,23 +41,28 @@ gcc -o genfilter build-scripts/gen_audio_resampler_filter.c -lm && ./genfilter >
 #include <stdio.h>
 #include <math.h>
 
+#ifndef M_PI
+    #define M_PI 3.14159265358979323846
+#endif
+
```

Comment:
We have `SDL_PI_D` defined elsewhere, with more digits, although I don't know if that makes a difference in practice:

```c
#define SDL_PI_D   3.141592653589793238462643383279502884
```

Reply:
double only has ~17 digits of precision so it wouldn't make a difference (though I did check to make sure).